Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml

```
@@ -836,6 +836,17 @@ script:
       description: User-provided notes related to the incident.
       type: string
     description: Returns additional details about a specific incident.
+  - name: asm-list-external-websites
+    arguments:
+    - name: authentication
+      description: authentication type string on which to search.
+    - name: limit
+      description: Maximum number of assets to return. The default and maximum is 100.
```

```suggestion
      description: Maximum number of assets to return. The maximum value is 100.
```
you do not want to put the default value in the description because when you generate the README, it is automatically added.